src/about-this-guide.md

```diff
@@ -10,8 +10,8 @@ There are six parts to this guide:
    useful no matter how you are contributing, about building, debugging,
    profiling, etc.
 2. [Contributing to `rustc`][p1-5]: Contains information that should be useful
-   no matter how you are contributing, about procedures for contribution,
-   stabilizing features, etc.
+   no matter how you are contributing, about procedures for contribution, using git
+   and github, stabilizing features, etc.
```

Comment:
```suggestion
   and GitHub, stabilizing features, etc.
```

I think it's OK to leave `git` lowercase though since there are many places like that already.